files/mongod.service

```
@@ -0,0 +1,30 @@
+[Unit]
+# We want the mongo service to be started very late in the boot
+# process.  This is because cloud-init may have to format the mongo
+# data, journal, and/or logs volumes, and that can take some time.
+#
+# multi-user.target is pretty late in the boot process, and including
+# cloud-final.service here should ensure that cloud-init is finished
+# running before this service is started.
+After=network.target multi-user.target cloud-final.service
+# Make sure all mongo file systems are mounted before starting.
+RequiresMountsFor=/var/lib/mongodb /var/lib/mongodb/journal /var/log/mongodb
+
```

These settings are added in the original logic so I believe they should be present here as well.

```suggestion

AssertPathIsMountPoint=/var/lib/mongodb
AssertPathIsMountPoint=/var/lib/mongodb/journal
AssertPathIsMountPoint=/var/log/mongodb
```

---

I dropped the commit where I removed this.  See commits 9cecedd, 714da15, and #33.